examples/post-sync/.dockerignore

```diff
@@ -0,0 +1,5 @@
+# Ignore binary
+sync-status-watch
```

Comment:
nit: suggest building the binary into an `_output/go/bin/<arch>/` dir, similar to how the parent build scripts work. Then you can ignore the whole _output dir here.